kas-fleet-manager/deploy-kas-fleet-manager.sh

```diff
@@ -196,8 +196,13 @@ deploy_kasfleetmanager() {
   echo "ADMIN_API_SSO_REALM=${ADMIN_API_SSO_REALM}" >> ${SERVICE_PARAMS}
 
   if [ -z "$( grep 'CLUSTER_LIST' $SERVICE_PARAMS || true; )" ]; then
-      echo "adding default CLUSTER_LIST to ${SERVICE_PARAMS}"
-      echo 'CLUSTER_LIST=[{"name": "'$(${OC} config view --minify --raw -o json | jq -r '.contexts[0].name')'","provider_type": "'${PROVIDER_TYPE}'","cluster_id": "'${DATA_PLANE_CLUSTER_CLUSTER_ID}'","cloud_provider": "'${DATA_PLANE_CLOUD_PROVIDER}'","region": "'${DATA_PLANE_CLUSTER_REGION}'","multi_az": true,"schedulable": true,"kafka_instance_limit": 5,"supported_instance_type": "standard,developer","status": "'${CLUSTER_STATUS}'","cluster_dns": "'${DATA_PLANE_CLUSTER_DNS_NAME}'"}]'  >> ${SERVICE_PARAMS}
+      if [ "${ENTERPRISE_ENABLED,,}" = "true" ] ; then
+        echo "ENTERPRISE ENABLED: adding EMPTY default CLUSTER_LIST to ${SERVICE_PARAMS}"
+        echo 'CLUSTER_LIST=[]'  >> ${SERVICE_PARAMS}
```

Comment:
We could alternatively check i the CLUSTER_LIST is set. If it is set in SERVICE_PARAMS, we do nothing. Otherwise, we set it to empty list. This allows for someone to define a list of managed clusters if they want to.

Reply:
That's basically what happens here. When not set in SERVICE_PARAMS we either make it an empty list for enterprise, or make it a default list based on the other configurations.

Reply:
+1 Thanks, I am from seeing it now.